R/fmelt.R

```diff
@@ -183,6 +183,13 @@ melt.data.table = function(data, id.vars, measure.vars, variable.name = "variabl
        verbose = getOption("datatable.verbose")) {
   if (!is.data.table(data)) stopf("'data' must be a data.table")
   if (missing(id.vars)) id.vars=NULL
+  id.sub = substitute(id.vars)
```

Comment:
I just copied the code for measure.vars and changed to id.vars -- do you think we should try a for loop instead? or is this ok?

Reply:
(for loop over measure.vars and id.vars to emphasize they are handled via the same logic)

Reply:
might be preferable to use a helper rather than copy-paste to ensure they stay in sync going forward